OMEdit/OMEditGUI/MainWindow.cpp

```diff
@@ -3238,11 +3263,13 @@ void MainWindow::createMenus()
   pViewWindowsMenu->addAction(mpThreeDViewerDockWidget->toggleViewAction());
 #endif
   pViewWindowsMenu->addAction(mpMessagesDockWidget->toggleViewAction());
+  pViewWindowsMenu->addAction(mpSearchDockWidget->toggleViewAction());
   pViewWindowsMenu->addAction(mpStackFramesDockWidget->toggleViewAction());
   pViewWindowsMenu->addAction(mpBreakpointsDockWidget->toggleViewAction());
   pViewWindowsMenu->addAction(mpLocalsDockWidget->toggleViewAction());
   pViewWindowsMenu->addAction(mpTargetOutputDockWidget->toggleViewAction());
   pViewWindowsMenu->addAction(mpGDBLoggerDockWidget->toggleViewAction());
+  pViewWindowsMenu->addAction(mpSearchDockWidgetAction);
```

Comment:
Don't create `mpSearchDockWidgetAction`. If you want to a add a shortcut then add this line,
`mpSearchDockWidget->toggleViewAction()->setShortcut(QKeySequence("Ctrl+h"));`
when you create `mpSearchDockWidget`.

If you look into the View->Windows menu there is an empty item right now.